### ISSUE TITLE
`/zh/` Broken Links Increased after merge.

### ISSUE THREAD

**Issue by butuzov** [kubernetes/website#18304]
**This is a Bug Report**

I do a small monitoring of my progress of removing deadlinks from kubernetes website, yesterday we had an increase of deadlinks from 187 to 295. All of the new urls are from `/zh/` path. I am not posting a list as it inefficient.

I have use my tool called [`deadlinks`](https://github.com/butuzov/deadlinks)

```
deadlinks kubernetes.io -n10 --no-colors --no-progress | grep /zh
```

**Comment by butuzov** [comment]
About 40 urls "fixed" in Chinese localization by `/zh` team.

**Comment by butuzov** [comment]
closed in favor of the kubernetes/website#18411
